Added config steps for Unraid (#29476)
home-assistant/home-assistant.github.io: source/_includes/installation/operating_system.md

    More details can be found about the command can be found [here](https://www.virtualbox.org/manual/ch08.html#vboxmanage-storageattach).

    </div>

> - title: Unraid

Reviewer: This has been scoped by an architectural decision, see:

<https://github.com/home-assistant/architecture/blob/master/adr/0015-home-assistant-os.md>

We don't want to support/add more than decided in there. If we want to extend this list, we need to discuss this in an architectural discussion and approve this first, before we add this to the documentation.

Feel free to open up an architectural proposal in our architecture repo.

../Frenck

Author: I was reading there and decided to add this to the docs based on:

> We will provide documentation for the following systems build on top of these technologies:
>  Proxmox (KVM/QEMU based)
>  Unraid (KVM/QEMU based)
>  ...

What am I missing?

Reviewer: You? Nothing. Me? A lot apparently 🙈

Author: Hehe… Super! 👍🏻
Is it too much to ask to get the hacktoberfest label back also? 🫣

  content: |
    1. Download the **.qcow2** image above and decompress it. (**Extract all** in Windows)
    2. Store the image in the **isos** share on your server.

>     3. Make sure "Settings" -> "VM manager" -> "Enable VMs" is enabled.

Reviewer 2: ```suggestion
    3. Make sure **Settings** > **VM manager** > **Enable VMs** is enabled.
```

>     3. Make sure **Settings** > **VM manager** > **Enable VMs** is enabled.

Reviewer 2: ```suggestion
    3. Make sure under **Settings** > **VM manager**, **Enable VMs** is enabled.
```

>     4. Create a new virtual machine: VMS -> ADD VM.

Reviewer 2: ```suggestion
    4. Create a new virtual machine: **VMS** > **Add VM**.
```

    5. Select type **Linux** and give the VM a name and a description.
    6. Select the CPU cores you want to let the VM use and give it some memory. 

>     7. Under **Primary vDisk Location** select *Manual* and then select the qcow2 image.

Reviewer 2: ```suggestion
    7. Under **Primary vDisk Location**, select **Manual** and then select the qcow2 image.
```

    8. Select your keyboard language under **VM Console Keyboard**.

>     9. Select *br0* under **Network Source**.

Reviewer 2: ```suggestion
    9. Select **br0** under **Network Source**.
```

>     10. Select *virtio* under **Network model**.

Reviewer 2: ```suggestion
    10. Select **virtio** under **Network model**.
```

>     11. Select any USB-devices that you want to pass through to Home Assistant, such as Zigbee- or Zwave-controllers.

Reviewer 2: ```suggestion
    11. Select any USB-devices that you want to pass through to Home Assistant, such as Zigbee- or Z-Wave controllers.
```

>     12. **Deselect** "Start VM after creation".

Reviewer 2: ```suggestion
    12. Deselect **Start VM after creation**.
```

>     13. Click **CREATE**.

Reviewer 2: ```suggestion
    13. Select **Create**.
```

>     14. Click on **the name** of your new VM and click on the capacity number for your disk. Here you can expand the disk to whatever your needs are. Default 32GB.

Reviewer 2: ```suggestion
    14. Select the name of your new VM and select the capacity number for your disk. Here, you can expand the disk to whatever your needs are. The default is 32&nbsp;GB.
```

>     15. Click on **the icon** of your new VM and select *start with console(VNC)*.

Reviewer 2: ```suggestion
    15. Select the icon of your new VM and select **start with console (VNC)**.
```
